modules/thunder_liveblog/thunder_liveblog.install

```diff
@@ -148,3 +150,28 @@ function thunder_liveblog_install() {
   }
 
 }
+
+/**
+ * Adds image to the list of available paragraphs.
+ */
+function thunder_liveblog_update_8101() {
+  /** @var \Drupal\thunder_updater\Updater $thunderUpdater */
+  $thunderUpdater = \Drupal::service('thunder_updater');
+
+  /** @var \Drupal\field\Entity\FieldConfig $field */
+  $field = FieldConfig::load('liveblog_post.liveblog_post.field_embed_media');
+
+  if ($field) {
```

Comment:
Handling should be more failure proof and please add logging for successes + fails. So only if `$field->save()` is successful -> mark checkbox and log success Additionally cases when save() fails or there is no field should be accordingly logged.